spec/lib/config_spec.rb

```diff
@@ -12,4 +13,15 @@
       expect(described_class.pantry_finder_api_url).to eq pantry_finder_api_url
     end
   end
+
+  context 'with facebook_api_url' do
+    before do
+      allow(described_class).to receive(:facebook_api_url)
+        .and_return(fb_api_url)
+    end
+
+    it 'returns url' do
+      expect(described_class.facebook_api_url).to eq fb_api_url
+    end
```

Comment:
This test doesn't test anything.
You've stubbed the response to the method and then just called the stubbed method.